### DISCUSSION TITLE
Add the ability to set ranking range of participants in playlists

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#22703] [Ideas]
This feature should be for osu!supporter owners only; that's the main reason why this idea has came to me.

**Comment by Axilotl17** [comment]
that seems like a dumb feature for osu!supporter. all of the osu supporter benefits are made to benefit the person with supporter, not limit what other users can do, necessarily. 

additionally, gatekeeping based on rank sounds like a *really* bad idea. And there are so many weird situations that could happen with this; what if a player enters the playlist above the threshold, then immediately drops down below it? what if someone is severely underranked, and they abuse that to play in the threshold? 

I just don't see the practical value of adding this, and gatekeeping players (especially new, low rank players) seems like such a mean thing to do.

**Reply by mk56-spn** [reply]
All of what you said applies to digit capped tournaments, I don't see the issue with allowing rank gatekeeping. Also the whole " someone losing rank and not being in the range after " doesn't matter since you can just have it only be checked once upon joining . If someone wants to set up a 3!3 star map lobby for 7 digits to have a little fun in why should they be forced to deal with bored 4 digits purposeful ruining it.

 Derankers will always exist but just by rank limiting you avoid most cases that would occur 

And it's hardly gatekeeping anymore than the existence of a private playlist or such would be. It should be up to the host of playlist to decide what sort of player they want to deal with. Not really the players business IMO

**Reply by Axilotl17** [reply]
okay, even if it is a good idea; why would it be a supporter only feature? additionally, digit capped tournaments are totally different from a multiplayer lobby?

>  If someone wants to set up a 3!3 star map lobby for 7 digits to have a little fun in why should they be forced to deal with bored 4 digits purposeful ruining it. 

ive been pushing for **map star rating** limits for awhile. that would also solve this issue, assuming you think more skilled players would add hard maps. sometimes 4 digits want to have fun playing easy maps, with other players, and it doesnt hurt 7 digits to have the 4 digit play with them? 

> Derankers will always exist but just by rank limiting you avoid most cases that would occur

you are right, most of that argument was useless. 

> And it's hardly gatekeeping anymore than the existence of a private playlist or such would be. It should be up to the host of playlist to decide what sort of player they want to deal with. Not really the players business IMO

i think there should be discretion to allowing only your friends to join and allowing only certain ranks to join. while of course players could do this manually, it doesn't feel like something we should support them in doing.

**Reply by mk56-spn** [reply]
I agree it shouldn't be a supporter thing 


> ive been pushing for **map star rating** limits for awhile. that would also solve this issue, assuming you think more skilled players would add hard maps. sometimes 4 digits want to have fun playing easy maps, with other players, and it doesnt hurt 7 digits to have the 4 digit play with them? 

How does it not? It means the number 1 on the playlist / multi leaderboard is effectively set in stone. If 4 digits want to have open 3 star multi lobbies nobodies stopping them from doing that. I just don't see why people should have to deal with their shenanigans just so they can have like 3 more lobbies available

**Reply by Axilotl17** [reply]
I think that there is already a "divide" between new and experienced players and this idea would do nothing but worsen that divide. and i disagree that it hurts people to play maps at their level with someone not at their level. if youre so concerned about it make a private room.

**Reply by mk56-spn** [reply]
> I think that there is already a "divide" between new and experienced players and this idea would do nothing but worsen that divide. and i disagree that it hurts people to play maps at their level with someone not at their level. if youre so concerned about it make a private room.

I feel you are missing the entire point. The idea behind this is to allow new players to actually enjoy multiplayer among themselves without having to tediously join closed lobbies if they so wish. Why should they be disallowed from such a thing. This is not a mandatory setting. Hence the idea that it will create barriers between new and experienced players is bankrupt for the most part. Those who wish to seek out lobbies with stronger players than themselves can. Those who don't should have an option to limit it.
